volatility/framework/plugins/windows/cmdline.py

```diff
@@ -77,7 +77,7 @@ def _generator(self, procs):
                     proc_id, exp.invalid_address, exp.layer_name)
     
             yield (0, (proc.UniqueProcessId, process_name, result_text))
-
+    
```

Comment:
It would be good to avoid committing whitespace additions like this if possible (I thought `yapf` would clear these out, but perhaps not).